index.html

```diff
@@ -134,56 +171,44 @@ <h2 class="contenthead">Professional Art Printing Data</h2>
                 <li>bootstrap</li>
                 <li>Ruby</li>
             </ul>
-            <button type="button" class="btn">See Projects</button>
-
-        </div>
+            <div>
+                <button type="button" class="bton">See Project</button>
             </div>
         </div>
+        </div>
+            </div>
 
-        <div class="project project5">
+        <div class="project6 project">
             <div class="content">
-                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <h2 class="contenthead">Data Dashboard Healthcare</h2>
                 <p class="contentp">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
             <ul class="projectlinks">
                 <li>html</li>
                 <li>bootstrap</li>
                 <li>Ruby</li>
             </ul>
-    
-            <button type="button" class="btn">See Projects</button>
-
-        </div>
+            <div>
+                <button type="button" class="bton">See Project</button>
             </div>
         </div>
+        </div>
 
-        <div class="project project6">
+        <div class="project7 project">
             <div class="content">
-                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <h2 class="contenthead">Website <br> Portfolio</h2>
                 <p class="contentp">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
             <ul class="projectlinks">
                 <li>html</li>
                 <li>bootstrap</li>
                 <li>Ruby</li>
             </ul>
-            <button type="button" class="btn">See Projects</button>
-
-        </div>
+            <div>
+                <button type="button" class="bton">See Project</button>
             </div>
-        </div> 
-        <div class="project project7">
-            <div class="content">
-                <h2 class="contenthead">Professional Art Printing Data</h2>
-                <p class="contentp">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-            <ul class="projectlinks">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
-            </ul>
-            <button type="button" class="btn">See Projects</button>
-
         </div>
             </div>
         </div>
+        </div>
         </section>
                                             <!--ABOUT SECTION-->
 <section class="Sectionaboutme">
```

Comment:
This section is almost done, just add some margin to the top, plus align and resize the image properly.

![image](https://user-images.githubusercontent.com/34779928/194695640-dfc26c30-13fb-4ee9-ae53-7d4c65b445e7.png)